Test/git-issues/git-issue-2672.dfy

```diff
@@ -0,0 +1,82 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:cs "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:js "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:go "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:java "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:py "%s" >> "%t"
+// RUN: %diff "%s.expect" "%t"
```

Comment:
Can you use `%testDafnyForEachCompiler %s` instead? :)

Reply:
Can you remind me how to generate the output with that flag?

Reply:
It should just be a single copy of the program's output, not including the "Dafny program verifier ..." line. So in this case just:

```
true true true true true true true true true
false false false false false false false false false
false false false false false false false false false
true true true true true true true true true
true true true true true true true true true
false false false false false false false false false
true true true
false false false
```

It occurs to me that having a flag on the `TestDafny` tool to generate the expect file would be super handy, let me know if I can add that. :)

Reply:
Thanks!  Unfortunately it doesn't correctly detect unsupported features, so it fails on this example; I've reported the error at https://github.com/dafny-lang/dafny/issues/2718 , and I'm currently preparing a PR.  We can update this test file once that PR is merged, but I don't think we need to block on it.